### ISSUE TITLE
[Bug] Picker组件的defaultValue属性在harmonyOS以及iOS上无法正常体现出默认值

### ISSUE THREAD

**Issue by Panlichen1996** [ant-design/ant-design-mobile-rn#1381]
### 💬 前提强调

- [X] I have searched the [issues](https://github.com/ant-design/ant-design-mobile-rn/issues) of this repository and believe that this is not a duplicate.

### 🙋 描述详情/重现步骤

```
<Picker data={datas} defaultValue={['42', '4201', '420111']}>
    <List.Item arrow="horizontal">{'省市选择'}</List.Item>
</Picker>
```

### 🔴 版本

9.8.0

### 💻 运行环境

iOS

### ⚫️ `npx react-native info`输出信息

System:
  OS: Windows 10 10.0.19045
  CPU: (8) ia32 Intel(R) Core(TM) i7-9700 CPU @ 3.00GHz
  Memory: 4.02 GB / 15.84 GB
Binaries:
  Node:
    version: 18.14.1
    path: D:\software\nodejs\node.EXE
  Yarn:
    version: 1.22.22
    path: D:\software\nodejs\node_global\yarn.CMD
  npm:
    version: 9.8.0
    path: D:\software\nodejs\npm.CMD
  Watchman: Not Found
SDKs:
  Android SDK:
    API Levels:
      - "17"
      - "18"
      - "28"
      - "30"
      - "33"
      - "34"
      - "34"
    Build Tools:
      - 29.0.1
      - 29.0.2
      - 30.0.2
      - 30.0.3
      - 33.0.0
      - 33.0.1
      - 34.0.0
    System Images:
      - android-33-ext4 | Google Play Intel x86_64 Atom
      - android-33-ext5 | Google Play Intel x86_64 Atom
      - android-33 | Google Play Intel x86_64 Atom
      - android-33 | Google APIs ATD Intel x86_64 Atom
      - android-34-ext8 | Google Play Intel x86_64 Atom
      - android-34 | Google Play Intel x86_64 Atom
    Android NDK: Not Found
  Windows SDK: Not Found
IDEs:
  Android Studio: Not Found
  Visual Studio: Not Found
Languages:
  Java: 17.0.3.1
  Ruby: Not Found
npmPackages:
  "@react-native-community/cli": Not Found
  react:
    installed: 18.2.0
    wanted: 18.2.0
  react-native:
    installed: 0.72.5
    wanted: 0.72.5
  react-native-windows: Not Found
npmGlobalPackages:
  "*react-native*": Not Found
Android:
  hermesEnabled: true
  newArchEnabled: false
iOS:
  hermesEnabled: Not found
  newArchEnabled: Not found

**Comment by 1uokun** [comment]
Duplicate of #1331

**Comment by 1uokun** [comment]
fixed in `5.3.0`
